#!/usr/bin/env python
PACKAGE = "connectivity_controller"
#import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("r_min", double_t, 0, "Radius min", .4, 0.0, 1.5)

gen.add("r_max", double_t, 0, "Radius max", .7, 0.0, 1.5)

exit(gen.generate(PACKAGE, "connectivity_controller", "ConnectivityVariables"))
